deploy/helm/templates/swo-agent-statefulset.yaml

```diff
@@ -50,7 +50,10 @@ spec:
           imagePullPolicy: {{ .Values.swoagent.image.pullPolicy }}
           env:
             - name: UAMS_CLIENT_ID_OVERRIDE_SOURCE_NAME
-              value: {{ .Values.cluster.uid }}
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "common.fullname" (tuple . "-common-env") }}
+                  key: CLUSTER_UID
```

Comment:
Use the environment variable for CLUSTER_UID, like we do in all other containers.